workshops/python_turtle/README.md

```diff
@@ -0,0 +1,352 @@
+---
+name: 'Python Turtle'
+description: 'Exciting Visuals with Python Turtle'
+author: '@YashKalbande'
+---
+
+
+# Exciting Visuals with Python Turtle
+
+
+## Introduction
+Welcome! my friends.  My name is Yash, and I am a technology enthusiast. Have you ever heard that you can create exciting visuals with help of Python?  My goal is to help you understand the python turtle library. With the help of the turtle library, we can create stunning visuals. First, I'm going to present to you the basics of the python turtle library. Then I'll share with you shapes like Triangle, Star, Hexagon. Finally, I'll ask you to create some more visual using ideas given in the end.
```

Comment:
Don’t introduce yourself—this should stand alone as a tutorial

Reply:
done.